vms/platformvm/client.go

```diff
@@ -172,9 +172,6 @@ type Client interface {
 	GetRewardUTXOs(context.Context, *api.GetTxArgs) ([][]byte, error)
 	// GetTimestamp returns the current chain timestamp
 	GetTimestamp(ctx context.Context) (time.Time, error)
-	// GetValidatorsAt returns the weights of the validator set of a provided subnet
-	// at the specified height.
-	GetValidatorsAt(ctx context.Context, subnetID ids.ID, height uint64) (map[string]uint64, error)
```

Comment:
`GetValidatorsAt()` wasn't getting used anywhere before ? Or was it just to be used on platformvm externally?

Reply:
I think it's only there to run the Avalanche node as a plugin itself.